_posts/pl/2021-09-31-zarzadzanie-stanem-aplikacji-frontendowej.md

```diff
@@ -21,20 +21,20 @@ tags:
 - react
 ---
 
-W miarę rozwijania złożonych aplikacji webowych ważnym i nieoczywistym zagadnieniem staje się projektowanie przepływu informacji pomiędzy komponentami. Często mamy do czynienia z wieloma źródłami danych - mogą to być na przykład najróżniejsze zewnętrzne serwisy, czy polecenia wprowadzane przez użytkownika. Podstawowym rozwiązaniem jest przekazywanie tych danych przy pomocy rozwiązań dostarczanych przez wykorzystywany framework (na przykład data binding w Angularze), jednak bardzo łatwo można wyobrazić sobie sytuację, w której informacje, z których chcemy skorzystać, znajdują się w zupełnie innej części aplikacji, a przed nami stoi zadanie „przepchania" ich aż do interesującego nas miejsca. Jest to oczywiście rozwiązanie nieskalowalne, trudne w utrzymaniu i niezbyt eleganckie.
+W miarę rozwijania złożonych aplikacji webowych ważnym i nieoczywistym zagadnieniem staje się projektowanie przepływu informacji pomiędzy komponentami. Często mamy do czynienia z wieloma źródłami danych - mogą to być na przykład najróżniejsze zewnętrzne serwisy, czy polecenia wprowadzane przez użytkownika. Podstawowym podejściem jest przekazywanie tych danych przy pomocy rozwiązań dostarczanych przez wykorzystywany framework (na przykład data binding w Angularze), jednak bardzo łatwo można wyobrazić sobie sytuację, w której informacje, z których chcemy skorzystać, znajdują się w zupełnie innej części aplikacji, a przed nami stoi zadanie „przepchania" ich aż do interesującego nas miejsca. Jest to oczywiście rozwiązanie nieskalowalne, trudne w utrzymaniu i niezbyt eleganckie.
```

Comment:
Poniżej propozycja zmiany 2 i 3 zdania przy założeniu, że zewn. serwisy i wprowadzone przez użytkownika to rozłączne podmioty, wykluczające się i nie stawiamy w takim przypadku przecinka przed czy + krótkie zdania na blogach zwłaszcza na wstępie lepiej się czyta.
_Często mamy do czynienia z wieloma źródłami danych. Mogą to być na przykład najróżniejsze zewnętrzne serwisy czy polecenia wprowadzane przez użytkownika._